templates/index.html

```diff
@@ -10,36 +10,37 @@
 		crossorigin="anonymous"></script>
 		<link rel="stylesheet" href="static/main.css">
     </head>
-    <body>
+    <body id = "wholepage">
+		<form method="POST" action="" >
 		<div id = "header">
 			<h1 id = "appname">Deelingual</h1>
-			Translate Page:
-			<select id="page_language">
-					<option>English</option>
-					<option>French</option>
+			<label id = "translate_page"for="page_language">{{word_translated_list[6]}}</label>
```

Comment:
It looks great.

Reply:
We can add different text format for app name to make looks special from other texts from the page.